alpha_automl/automl_manager.py

```diff
@@ -67,31 +67,35 @@ def _search_pipelines(self, automl_hyperparams):
         found_pipelines = 0
 
         while True:
-            result = queue.get()
+            try:
+                result = queue.get(timeout=30)
+            except Exception:
```

Comment:
Shouldn't this line catch `queue.Empty` instead of `Exception`? The documentation says `queue.get()` throws `queue.Empty`.